### PR TITLE
🚧 Initial expansion on the point/measure tables 🚧

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

File: autocnet/io/db/model.py [hunk 1]
```diff
@@ -258,19 +258,46 @@ class PointType(enum.IntEnum):
...
+    datetime = Column(DateTime)
+    editLock = Column(Boolean)
+    ignore = Column(Boolean)
+    jigsawrejected = Column(Boolean)
+    referenceindex = Column(Integer)
+    # Our own, is this point live field
     active = Column(Boolean, default=True)
+    
+    # Provenance
+    apriori_surfacepointsource = Column(Integer)
```
Comment (jessemapel) [inline autocnet/io/db/model.py]: If we map point types from enumerations (ints) to strings, we should map these enumerations too

**Comment by jlaura** [comment]
@acpaquette Is your notebook based off of this?

**Comment by acpaquette** [comment]
@jlaura No, those two notebooks should work with just autocnet dev and plio master with some potential local changes :grimacing:

**Comment by jlaura** [comment]
@all this needs to wait until we have a decision about BCBF vs. lat/lon.

**Comment by jlaura** [comment]
I am going to leave this until #347 and #349 are merged. Then this can get updated, reviewed, and merged, to be able to round trip a full cnet.

**Comment by jessemapel** [comment]
@jlaura Is this still a thing we want to do? Should it just be closed?

**Pull request closed without merging**